src/Feature/Configuration/Feature_Configuration.re

```diff
@@ -333,8 +333,22 @@ module Commands = {
     );
 };
 
+module Keybindings = {
+  open Feature_Input.Schema;
+
+  let command = Commands.openConfigurationFile.id;
+
+  let openConfigOnMac =
+    bind(~key="<D-,>", ~condition="isMac" |> WhenExpr.parse, ~command);
```

Comment:
I believe the condition is case-sensitive (which may be a bug itself) - flipped from `IsMac` -> `isMac`.